src/v3/src/hooks/useHtmlContentParser.tsx

```diff
@@ -23,7 +23,7 @@ import preact, { h } from 'preact';
 
 export const useHtmlContentParser = (
   content: string | undefined,
-  options?: HTMLReactParserOptions,
+  options?: HTMLReactParserOptions & { variant?: 'monochrome'; },
```

Comment:
This is a little bit smelly, we might want to think a little more (but not necessarily block this work) about how to do this without drilling the prop that doesn't really have anything superficially to do with this hook

Reply:
I agree, initially what I thought I could do was just set the attribute on the link before we pass it to `loc`. What happens though is that the Parser doesn't recognize the `variant` attribute and strips it out :(

Reply:
I found another way around this issue. I think this approach is a little cleaner and re-usable for other scenarios.
https://github.com/okta/okta-signin-widget/pull/3110/commits/d2c2ad7ba73fef0ee61392a819d90a54b614e4a2